ipatests/test_xmlrpc/test_pwpolicy_plugin.py

```diff
@@ -186,28 +282,32 @@ def test_b_pwpolicy_add(self):
     def test_c_pwpolicy_find(self):
         """Test that password policies are sorted and reported properly"""
         result = api.Command['pwpolicy_find']()['result']
-        assert len(result) == 4
+        assert len(result) == 6
```

Comment:
Instead of modifying `test_c_pwpolicy_find` and `test_c_pwpolicy_find_pkey_only` because there are additional policies, you can simply delete the pwpolicies created in `test_maxlife_pwpolicy` at the end of `test_maxlife_pwpolicy`.